docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -2297,19 +2297,22 @@ DATEADD(DAY, 7, timestamp'2008-02-29 00:00:00')
 == DATEDIFF Function
 
 The DATEDIFF function returns the integer value for the number of
-_datepart_ units of time between _startdate_ and _enddate_. If
-_enddate_ precedes _startdate_, the return value is negative or zero.
+_datepart_ units of time between _startdate_ and _enddate_. 
 
 DATEDIFF is a {project-name} SQL extension.
 
 ```
 DATEDIFF (datepart, startdate, enddate)
 ```
 
+<<<
+[[syntax_description_of_datediff]]
+=== Syntax Description of DATEDIFF 
+
 * `datepart`
 +
 is YEAR, MONTH, DAY, HOUR, MINUTE, SECOND, QUARTER, WEEK, or one of the
-following abbreviations:
+following abbreviations, should be enclosed in quotation marks:
```

Comment:
I am not sure this is correct. I tried using MI, for example, and found that with quotes I get a syntax error, but without quotes, it works. I am researching this issue.

Reply:
Finished research. Verified that the abbreviations do not take quotes.

Reply:
Thanks Dave, for pointing out this mistake, I've corrected. :)